### PR TITLE
Fix make lint warnings

### PR DESCRIPTION
**Pull request by b00ris**
copy of https://github.com/status-im/status-go/pull/388 with fixes made from local repository.
- fixed make lint warnings
- cleared linter_exclude_list.txt
- removed some commented code
- made from local repository
- had to be done anew after https://github.com/status-im/status-go/pull/375
- fixed comments from https://github.com/status-im/status-go/pull/388

### CODE REVIEW

File: cmd/statusd/library.go [hunk 1]
```diff
@@ -88,52 +92,24 @@ func ValidateNodeConfig(configJSON *C.char) *C.char {
...
-func PopulateStaticPeers() *C.char {
-	err := fmt.Errorf("%v: %v", common.ErrDeprecatedMethod.Error(), "PopulateStaticPeers")
-	return makeJSONResponse(err)
-}
-
-//export AddPeer
-func AddPeer(url *C.char) *C.char {
-	err := fmt.Errorf("%v: %v", common.ErrDeprecatedMethod.Error(), "AddPeer")
-	return makeJSONResponse(err)
-}
```
Comment (tiabc) [inline cmd/statusd/library.go]: Why have you removed all these?

File: cmd/statusd/utils.go [hunk 2]
```diff
@@ -1355,9 +1358,6 @@ func startTestNode(t *testing.T) <-chan struct{} {
 			t.Log("Node started, but we wait till it be ready")
 		}
 		if envelope.Type == signal.EventNodeReady {
-			// manually add static nodes (LES auto-discovery is not stable yet)
-			PopulateStaticPeers()
```
Comment (tiabc) [inline cmd/statusd/utils.go]: Why?

File: e2e/api/api_test.go [hunk 3]
```diff
@@ -149,16 +145,16 @@ func (s *APITestSuite) TestCellsRemovedAfterSwitchAccount() {
 	require.NoError(err)
 
 	for i := 0; i < itersCount; i++ {
-		_, err := s.api.JailManager().NewCell(getChatId(i))
-		require.NoError(err)
+		_, e := s.api.JailManager().NewCell(getChatId(i))
+		require.NoError(e)
```
Comment (tiabc) [inline e2e/api/api_test.go]: Why `e`, not `err`? :) The last is common.

File: e2e/api/backend_test.go [hunk 4]
```diff
@@ -94,22 +94,26 @@ func (s *APIBackendTestSuite) TestRaceConditions() {
...
+			// CreateChildAccount
+			log.Info("CreateChildAccount()")
+			address, pubKey, err = s.Backend.AccountManager().CreateChildAccount(address, "password")
+			s.T().Logf("CreateAccount(), error: %v (address: %v, pubKey: %v)", err, address, pubKey)
+
+			// RecoverAccount
+			log.Info("RecoverAccount()")
+			address, pubKey, err = s.Backend.AccountManager().RecoverAccount("password", mnemonic)
+			s.T().Logf("RecoverAccount(), error: %v (address: %v, pubKey: %v)", err, address, pubKey)
+
```
Comment (tiabc) [inline e2e/api/backend_test.go]: Could you elaborate?

File: geth/jail/doc.go [hunk 5]
```diff
@@ -1,88 +1,85 @@
...
+//	cell.Set("__captureSuccess", func(res otto.Value) { ... })
+//
+//	cell.Run(`fetch('http://example.com').then(function(r) {
+//		return r.text()
+//	}).then(function(data) {
+//		// user code
+//		__captureSuccess(data)
+//	}))
+//
+//
```
Comment (tiabc) [inline geth/jail/doc.go]: This thing is auto-generated. @divan :) Worth excluding from linting.

File: e2e/api/api_test.go (hunk 3 above)
Comment (b00ris) [reply e2e/api/api_test.go]: This code contains shadowing variable.

File: geth/jail/doc.go (hunk 5 above)
Comment (b00ris) [reply geth/jail/doc.go]: Got it.

File: cmd/statusd/utils.go (hunk 2 above)
Comment (b00ris) [reply cmd/statusd/utils.go]: I've remove this method. I think it's unuseful.
{code}
-func PopulateStaticPeers() *C.char {
 -	err := fmt.Errorf("%v: %v", common.ErrDeprecatedMethod.Error(), "PopulateStaticPeers")
 -	return makeJSONResponse(err)
 -}
{code}

File: e2e/api/api_test.go (hunk 3 above)
Comment (tiabc) [reply e2e/api/api_test.go]: And the same reason for other places with this change?

File: cmd/statusd/utils.go (hunk 2 above)
Comment (tiabc) [reply cmd/statusd/utils.go]: Ok, good decision.

File: e2e/api/backend_test.go (hunk 4 above)
Comment (b00ris) [reply e2e/api/backend_test.go]: Yes, it helps me to avoid 3th err shadowing using one err vaiable.

File: cmd/statusd/library.go (hunk 1 above)
Comment (b00ris) [reply cmd/statusd/library.go]: It's deprecated method,  Dead code linter pointed to some of them. I think i could to delete this deprecated method. Should i return them?

Comment (tiabc) [reply cmd/statusd/library.go]: Already answered below. Nope, it was good decision.

File: e2e/api/api_test.go (hunk 3 above)
Comment (b00ris) [reply e2e/api/api_test.go]: Yes, same reason.

**Comment by b00ris** [comment]
@tiabc returned geth/jail/doc.go and exclude it from linters check

**Comment by tiabc** [comment]
Thanks, awesome work!

**Pull request merged**